mobile/src/main.jsx

```diff
@@ -56,28 +58,46 @@ document.addEventListener('DOMContentLoaded', () => {
       saveState(stateToBeSaved)
     })
 
-    function requireSetup (nextState, replace) {
-      const url = store.getState().mobile.settings.serverUrl
+    initClient(store.getState().mobile.settings.serverUrl)
+
+    function requireSetup (nextState, replace, callback) {
+      const client = store.getState().mobile.settings.client
       const isSetup = store.getState().mobile.settings.authorized
-      if (!isSetup) {
+      if (isSetup) {
+        isClientRegistered(client).then(clientIsRegistered => {
+          if (clientIsRegistered) {
+            const options = {
+              onError: (err) => {
+                console.log('on error fron the client', err)
+                console.warn(`Your device is no more connected to your server: ${store.getState().mobile.settings.serverUrl}`)
+                store.dispatch(revokeClient())
+              }
+            }
+            cozy.client.offline.startRepeatedReplication('io.cozy.files', 15, options)
```

Comment:
@nono I added this callback (`options.onError`) to `cozy.client.offline` to be warned when the scheduled replication fails. In this PR https://github.com/cozy/cozy-client-js/pull/102 I added the requisite change to handle this case.